pypesto/sampling/pymc3.py

```diff
@@ -10,9 +10,19 @@
 logger = logging.getLogger(__name__)
 
 try:
-    import pymc3 as pm
-    import theano.tensor as tt
+    from theano.tensor import Op, dvector, dscalar
     from theano.gof.null_type import NullType
+except ImportError:
```

Comment:
@dweindl Do you know how to best do this? Here we have Op as a base class, so we need to have _something_ to properly define the `TheanoLogProbability` without errors, which otherwise fails when running any (non pymc3 related) pypesto routine due to inclusion in `__init__`. Alternatively, one could catch the import of `Pymc3Sampler` in `__init__` maybe ...

Reply:
> catch the import of Pymc3Sampler in __init__ maybe

Nothing if this file is usable if that import fails? Then this sounds reasonable.